catalogue/webapp/components/RequestingDayPicker/RequestingDayPicker.tsx

```diff
@@ -35,6 +34,187 @@ type DayPickerWrapperProps = {
 };
 
 const DayPickerWrapper = styled.div<DayPickerWrapperProps>`
+  .DayPicker {
+    display: inline-block;
+    font-size: 1rem;
+  }
+  .DayPicker-wrapper {
+    position: relative;
+    -webkit-flex-direction: row;
+    -moz-box-orient: horizontal;
+    -moz-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    padding-bottom: 1em;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  .DayPicker-Months {
+    display: -webkit-flex;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-wrap: wrap;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: center;
+    -moz-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+  }
+  .DayPicker-Month {
+    display: table;
+    margin: 1em 1em 0;
+    border-spacing: 0;
+    border-collapse: collapse;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
```

Comment:
You can remove these (and equivalent in the other bits) as [styled-components does autoprefixing](https://styled-components.com/docs/basics#motivation)

Reply:
ah good point, I didn't actually look at what the styles were... thanks